hwdb.d/60-evdev.hwdb

```diff
@@ -482,11 +482,16 @@ evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnHP:pnHPENVYx360Convertible15-cn0xx
 # HUION
 #########################################
 
-# HUION Inpiroy H420X
+# HUION Inspiroy H420X
 evdev:input:b0003v256Cp0064*
  EVDEV_ABS_00=::200
  EVDEV_ABS_01=::200
 
+# HUION Inspiroy 2 L
+evdev:input:b0003v256Cp0068*
+ EVDEV_ABS_00=::200
+ EVDEV_ABS_01=::200
```

Comment:
This is the same data as the other huion entry. Please merge the two. See how this is done for the immediately preceeding entry for example.